### PR TITLE
explaining the API migration in CHANGELOG

### PR DESCRIPTION
**Pull request by caesarxuchao**
Fix https://github.com/kubernetes/client-go/issues/238.
Fix https://github.com/kubernetes/client-go/issues/238.

### CODE REVIEW

**Comment by lavalamp** [comment]
LGTM

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,7 +2,18 @@ TODO: This document was manually maintained so might be incomplete. The
 automation effort is tracked in
 https://github.com/kubernetes/client-go/issues/234.
 
-# HEAD (changes that will go into the next release)
+# HEAD (changes that will go into v5)
+
+** Breaking changes**
+
+pkg/api and pkg/apis are moved to
+[k8s.io/aip](https://github.com/kubernetes/api). Other kubernetes repositories
```
Comment (ash2k) [inline CHANGELOG.md]: typo

Comment (caesarxuchao) [inline CHANGELOG.md]: Eh, I stared at the line 30 seconds and found the typo. Will send a patch.

Comment (caesarxuchao) [inline CHANGELOG.md]: Fixed by https://github.com/kubernetes/client-go/pull/245

**Pull request merged**